app/src/main/java/net/squanchy/service/firestore/injection/FirestoreModule.kt

```diff
@@ -11,16 +10,12 @@ import net.squanchy.service.firestore.FirestoreDbService
 class FirestoreModule {
 
     @Provides
+    @ApplicationLifecycle
     internal fun firebaseFirestore(): FirebaseFirestore {
-        return FirebaseFirestore.getInstance().apply {
-            val settings = FirebaseFirestoreSettings.Builder(firestoreSettings)
-                .setPersistenceEnabled(false)
-                .build()
-            firestoreSettings = settings
-        }
+        return FirebaseFirestore.getInstance()
```

Comment:
Any reason why we need to remove the offline persistence?

Reply:
We are adding it not removing it